packages/block-editor/src/components/grid/style.scss

```diff
@@ -178,7 +178,7 @@
 		@include break-small() {
 			height: $grid-unit-50;
 			position: relative;
-			top: -5px; // Should be -4px, but that causes scrolling when focus lands on the movers, in a 60px header.
+			top: #{$grid-unit-05 * -1};
```

Comment:
Should we retain the comment, so that folks know the reason for the `-1px` ?

Reply:
I've made it even clearer by 1bf30376c1bde9d6960a3edb65f41fdab28afe84 what the intent of these styles is, including the top property.

Reply:
Can use perhaps $border-width instead of 1px, that might imply the comment.

Reply:
> $border-width instead of 1px

Here 1px is not used. `#{$grid-unit-05 * -1}` becomes `-4px`. Am I missing something?